source/reference/access-mdb-shell-help.txt

```diff
@@ -0,0 +1,241 @@
+
+       Runs an arbitrary command against the admin database.
+
+       For more information on usage: https://docs.mongodb.com/manual/reference/method/db.adminCommand
+
+- To see additional usage details for a database method in ``mongosh``,
+  type the ``db.<method name>`` without the parenthesis (``()``). The
+  following example returns details about the
+  :method:`db.adminCommand()` method:
+
+  .. code-block:: javascript
+
+     db.adminCommand
+
+  The output resembles the following:
+
+  .. code-block:: javascript
+     :copyable: false
+
+     [Function] {
+       serverVersions: [ '3.4.0', '4.4.0' ],
+       returnsPromise: true,
+       topologies: [ 0, 2, 1 ],
+       returnType: { type: 'unknown', attributes: {} },
+       platforms: [ 0, 1, 2 ],
+       help: [Function] Help
+     }
+
+
+.. _mdb-shell-help-collection:
+
+Collection Help
+---------------
+
+In ``mongosh``:
+
+- To see the list of collections in the current database, use the
+  ``show collections`` command:
+
+  .. code-block:: javascript
+
+     show collections
+  
+- To see the help for methods available on the collection objects
+  (e.g. ``db.<collection>``), use the ``db.<collection>.help()``
+  method:
+
+  .. include:: /includes/admonitions/help-interchangeable.rst
+
+  .. code-block:: javascript
+
+     db.collection.help()
+
+  ``<collection>`` can be the name of a collection that exists,
+  although you may specify a collection that doesn't exist.
+
+  The output resembles the following:
+
+  .. code-block:: none
+     :copyable: false
+
+     Collection Class:
+
+       aggregate               Calculates aggregate values for the
+                               data in a collection or a view.
+       bulkWrite               Performs multiple write operations
+                               with controls for order of execution.
+       count                   Returns the count of documents that
+                               would match a find() query for the
+                               collection or view.
+       countDocuments          Returns the count of documents that
+                               match the query for a collection or view.
+       deleteMany              Removes all documents that match the
+                               filter from a collection.
+       deleteOne               Removes a single document from a
+                               collection. 
+       distinct                Finds the distinct values for a specified
+                               field across a single collection or view
+                               and returns the results in an array.
+       estimatedDocumentCount  Returns the count of all documents in a
+                               collection or view.
+       find                    Selects documents in a collection or view.
+       
+       ...
+
+- To see help for a specific collection method in ``mongosh``, type the
+  ``db.<collection>.<method name>`` without the parenthesis (``()``),
```

Comment:
Similar comment above re "without the parens".

Reply:
updated